docs/languages/csharp/expression-trees.md

```diff
@@ -0,0 +1,66 @@
+﻿# Expression Trees
+
+If you have used LINQ, you have experience with a rich library
+where `Func` and `Func<T>` are part of the API. (If you are not familiar
+with LINQ, you probably want to read [that tutorial](linq.md) and
+the tutorial on [lambda expressions](lambda-expressions.md) before this one.)
+*Expression Trees* provide richer interaction with the arguments that
+are functions.
+
+You write function arguments, typically using Lambda Expressions, when
+you create LINQ queries. In a typical LINQ query, those function arguments are
+transformed into a delegate the compiler creates. 
+
+When you want to have a richer interaction, you need to use *Expression Trees*.
+Expression Trees represent code as a structure that you can examine,
+modify, or execute. These tools give you the power to manipulate code during
+runtime. You can write code that examines running algorithms, or injects new
```

Comment:
You probably wanna use run time here. 

Use run time to indicate the period of time during which a program is running. 
Use runtime as an abbreviation for common language runtime or, in a more generic sense, to indicate an active program during run time. 
Use run-time as an adjective. 

The runtime performs run-time functions during run time.